includes/managers/icons.php

```diff
@@ -445,10 +447,10 @@ public function ajax_enable_svg_uploads() {
 	 * Icons Manager constructor
 	 */
 	public function __construct() {
-		if ( is_admin() ) {
-			// @todo: remove once we deprecate fa4
-			add_action( 'elementor/admin/after_create_settings/' . Settings::PAGE_ID, [ $this, 'register_admin_settings' ], 100 );
-		}
+		// @todo: remove once we deprecate fa4
+		// Previously this action were called only when `is_admin() equals true`, but the tracker not always run as admin.
+		// and this data is needed.
+		add_action( 'elementor/admin/after_create_settings/' . Settings::PAGE_ID, [ $this, 'register_admin_settings' ], 100 );
```

Comment:
You can call `register_admin_settings` directly

Reply:
Handled.